action.yml

```diff
@@ -121,6 +125,21 @@ runs:
             TWINE_OPTS="$TWINE_OPTS --skip-existing"
           fi
           python -m pip install twine
+          if [[ -z "$TWINE_PASSWORD" ]]; then
+            # vis https://docs.pypi.org/trusted-publishers/using-a-publisher/#the-manual-way
+            export TWINE_PASSWORD=$(
+        python -c 'import json, os, urllib.request as req
+        audience = req.urlopen(os.getenv("INPUT_REGISTRY_DOMAIN") + "/_/oidc/audience").read()
+        token = json.load(req.urlopen(req.Request(
+            url=os.getenv("ACTIONS_ID_TOKEN_REQUEST_URL"),
+            data=audience,
+            headers={"Authorization": "bearer " + os.getenv("ACTIONS_ID_TOKEN_REQUEST_TOKEN")},
+            method="GET")))
+        print(json.load(req.urlopen(req.Request(
+            url=os.getenv("INPUT_REGISTRY_DOMAIN") + "/_/oidc/github/mint-token",
+            data=json.dumps({"token": token["value"]}).encode(),
+            method="POST")))["token"], end="")')
```

Comment:
getting `urllib.error.HTTPError: HTTP Error 422: Unprocessable Entity` with this